ci-operator/step-registry/hypershift/azure/create/hypershift-azure-create-chain.yaml

```diff
@@ -84,6 +84,9 @@ chain:
     - name: HYPERSHIFT_OLM_CATALOG_PLACEMENT
       default: ""
       documentation: "OLM Catalog Placement for the HostedCluster. Options: management, guest. Defaults if empty. "
+    - name: HYPERSHIFT_MANAGED_SERVICE
```

Comment:
I see that the `hypershift-azure-create-chain` is used in multiple other chains and workflows, which probably don't
define this new `HYPERSHIFT_MANAGED_SERVICE` env var. This means they all will have this new behavior by default once
this PR merges. Is that deliberate? Can you explain why?